MH-FiorellaTheme5/source/javascripts/site.js

```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
+    document.getElementById("mySidenav").style.width = "95%";
+    document.getElementById("logo").src = "../images/home/logo_fiorella_menu.png";
```

Comment:
Why are you using javascript to set the src logo image? Please remove this as it is a bad practice. 
Or give me an explanation on why you are using this?